bears/general/IndentationBear.py

```diff
@@ -330,7 +330,10 @@ def get_unspecified_block_range(self,
                                 un-indent specifiers.
         """
         specifiers = list(self.get_valid_sequences(
-            file, indent_specifier, annotation_dict))
+            file,
+            indent_specifier, annotation_dict,
```

Comment:
if you are going with this format, i agree with @fneu , should be newline after each param